doc/contributing/advocacy-ambasador-program.md

```diff
@@ -0,0 +1,81 @@
+
+The project aims to support and encourage developers and developer
+advocates to promote Node.js as a project and to help communicate
+key messages to their constituencies.
+
+The ambassador program does that by:
+
+* Maintaining an up to date set of messages that it would like abassadors
+  to promote. These will be documented in the nodejs/ambassadors repository.
+* Maintaining an up to date set of topics on which additional content
+  would benefit the community. These will be documented in the
+  nodejs/ambassadors repository.
+* Providing a space for collaborators to share new topics/features they
+  are working on, when they would be happy for ambassadors to help share and get
+  early feedback.
+* Helping ambassadors promote content that aligns with the
+  messages and topics defined through social media and other channels.
+* Advocating for ambasadors to be part of the OpenJS speakers bureau, even if the
+  ambassador is not otherwise and active member of the project itself.
+
+## Abassadors nominations
+
+Any member of the Node.js organization may nominate an ambassador.
+The Technical Steering Committee (TSC) will review/approve ambassadors.
+
+In the initial phase of the program:
+
+* there will be a maximum of 4 ambassadors
+* the term will be for the calendar year. After a year ambassadors automatically
+  drop out of the ambassador list and must be re-nominated
+* ambassadors will be documented on the nodejs/ambassadors README.md
+
+The TSC may chose to remove an ambassador if things are not working
+out in terms of the content promoted and alignment with the project.
+
+Members of the Node.js organization can nominate ambasssadors by opening
+an issue in: TBD.
+
+## Reviewing content
+
+Ambassadors are asked to share links to new content before asking
+for the content to be promoted. The goal is to provide the opportunity
+for collaborators to help improve content as opposed to a detailed
+nitpick review.
+
+This is done by opening an issue in the nodejs/ambassadors
+repository on which access is limited to members of the Node.js
+organization. The issue may include the link to existing published
+content or a draft of the content itself.
+
+Interested members of the Node.js organization may comment/make
+suggestions in the issue and the ambassadors are expected to
+consider those suggestions and update the content when they feel it
+is appropriate.
+
+Explicit approval is not required in the issue (ie if there is zero
+input that's ok), but objections should addressed before
+requesting promotion.
+
+## Promoting content
+
+We trust ambassadors to only request promotion of content that lines
+up with the messages and topics outlined, that this content does not overly
+market their employer or product/business, and that they have followed the review
+process. An abassador can request promotion of content in the following ways:
+
+* Requesting that a tweet be retweeted by the Node.js twitter account.
```

Comment:
Should we say the Node.js Twitter account or X account?

Reply:
```suggestion
* Requesting that a tweet be retweeted by the Node.js X account.
```